CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # OHHTTPStubs — CHANGELOG
 
+## [6.1.1](https://github.com/AliSoftware/OHHTTPStubs/releases/tag/6.1.1)
```

Comment:
Please just use a new `## Master` section for now instead of a future version number as we don't know when we'll release a new version and what its number will be yet.

(besides, your PR _adds_ a feature, while bumping the 3rd number of a version means fixing a bug and me feature requires bumping the minor version number not the patch —read semver.org to learn more if you like)

Reply:
Thanks, will update